### ISSUE TITLE
Screen reader is narrating ‘AIRT’ as a word instead of narrating as abbreviation.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5019]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Visually challenged users who rely on screen readers are getting impacted if Screen reader is narrating ‘AIRT’ as a word instead of narrating as abbreviation.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the URL: https://airtuserexpdev.azurewebsites.net/Dashboard in Edge Browser->SAGE chat window will display.
Step 2: Turn on NVDA and navigate through the page.
Step 3: Verify whether Screen reader is narrating ‘AIRT’ as a word instead of narrating as abbreviation or not.

### What do you expect?

NVDA should announce ‘AIRT’ as an abbreviation instead of narrating as word.

### What actually happened?

Screen reader is narrating ‘AIRT’ as a word instead of narrating as abbreviation.
Since screen reader is pronouncing as a word, user would not understand the meaning and would miss out the significance of the text.

### Do you have any screenshots or recordings to repro the issue?

![WCAG 3 1 4 - Screen reader is narrating ‘AIRT’ as a word instead of narrating as abbreviation](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/ab1ab81f-46cf-4d63-97a9-f670e02c22c0)

### WCAG reference

https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645906](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645906)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
Bot author should change the text in a way that the screen reader will say A.I.R.T. Web Chat has no control on the message.
